libkbfs/state_checker.go

```diff
@@ -320,18 +320,12 @@ func (sc *StateChecker) CheckMergedState(ctx context.Context, tlf TlfID) error {
 			if !ok {
 				sc.log.CDebugf(ctx, "Bad block server: %T", jbs.BlockServer)
 			}
-			// For now skip state checking until KBFS-1439 is fixed,
```

Comment:
Hrmm, apparently not.  According to the log, the type of `jbs.BlockServer` is still `libkbfs.journalBlockServer`.  How do I get the type of the embedded BlockServer?